site-scrapers/UMassAmherst/index.js

```diff
@@ -22,14 +22,14 @@ async function ScrapeWebsiteData(browser) {
     await page
         .waitForSelector(".loadingSpinner", { hidden: true })
         .catch(() => {});
+    // Wait for the buttons to show up
+    await page.waitForSelector(".slds-button").catch(() => {});
 
     const content = await page.content();
 
     const result = {
         hasAvailability:
-            content.indexOf(
-                "There are currently no time slots available for vaccinations."
-            ) == -1,
+            content.indexOf("there are no time slots available") == -1,
```

Comment:
A little nitpicky but could we just do "no time slots available" since that seems to be common text?

Reply:
Now... "Sorry there are no time slots available at the moment to book first and second dose appointments, please check back later"